app/views/books/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>New Book</h1>
+<%= form_for @book do |f| %>
+  <% if flash[:notice] %>
+    <p><%= flash[:notice] %></p>
+  <% end %>
+  <%= f.label :title%>
+  <%= f.text_field :title, type: :name, required: true %>
+  <br>
+  <%= f.label :year_published %>
+  <%= f.text_field :year_published, type: :number, min: -4000, max: 2019, required: true%>
+  <br>
+  <%= f.label :page_count %>
+  <%= f.text_field :page_count, type: :number, min: 1, required: true %>
+  <br>
+  <%= f.label :thumbnail %>
```

Comment:
Consider changing the label to say something like "Thumbnail URL", since the user doesn't actually upload an image file.